File: intro/abcz2.c
#include <stdio.h>

int main() {
    int a;
    int toggle = 1;
    
    scanf("%d", &a);
    
    if ( a % 2 == 0 ) {
        printf("alpha\n");
        toggle = 0;
    }
    if ( a % 3 == 0 ) {
        printf("bravo\n");
        toggle = 0;
    }
    if ( a % 5 == 0 ) {
        printf("charlie\n");
        toggle = 0;
    }
    if ( toggle ) {
        printf("zulu\n");
    }
    
    return 0;
}

File: bc-w4/php/abcz2.php
<?php 
    $a = intval(fgets(STDIN));
    $toggle = 1;
    
    if ( $a % 2 == 0 ) {
        echo "alpha" . PHP_EOL;
        $toggle = 0;
    }
    if ( $a % 3 == 0 ) {
        echo "bravo" . PHP_EOL;
        $toggle = 0;
    }
    if ( $a % 5 == 0 ) {
        echo "charlie" . PHP_EOL;
        $toggle = 0;
    }
    if ( $toggle ) {
        echo "zulu" . PHP_EOL;
    }
?>
